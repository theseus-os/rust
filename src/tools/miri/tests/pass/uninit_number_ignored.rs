//@compile-flags: -Zmiri-allow-uninit-numbers
// This test is adapted from https://github.com/rust-lang/miri/issues/1340#issue-600900312.

fn main() {
    let _val1 = unsafe { std::mem::MaybeUninit::<usize>::uninit().assume_init() };
    let _val2 = unsafe { std::mem::MaybeUninit::<i32>::uninit().assume_init() };
    let _val3 = unsafe { std::mem::MaybeUninit::<f32>::uninit().assume_init() };
}
