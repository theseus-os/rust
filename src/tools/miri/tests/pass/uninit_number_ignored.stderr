WARNING: `-Zmiri-allow-uninit-numbers` is deprecated and planned to be removed. Please let us know at <https://github.com/rust-lang/miri/issues/2187> if you rely on this flag.
